pptx/oxml/text.py

```diff
@@ -459,11 +471,43 @@ class CT_TextParagraphProperties(BaseOxmlElement):
     lnSpc = ZeroOrOne("a:lnSpc", successors=_tag_seq[1:])
     spcBef = ZeroOrOne("a:spcBef", successors=_tag_seq[2:])
     spcAft = ZeroOrOne("a:spcAft", successors=_tag_seq[3:])
+    eg_textBulletColor = ZeroOrOneChoice(
```

Comment:
Can you change these to be only camel case instead of having a mix of camel and snake case?

Reply:
This is consistent with how things are named throughout the rest of the library for element groups.  Fortunately, they aren't exposed to the end user of the library, just used internally.